user-group/announcement-01.html

```diff
@@ -0,0 +1,101 @@
+                                            <table style="border-spacing:0;" width="100%">
+                                                <tbody>
+                                                    <tr>
+                                                        <td style="text-align:center;padding-top:30px;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;">
+                                                            <h3 style="font-family:sans-serif;font-weight:bold;text-align:center;font-size:22px;line-height:28px;Margin-top:0;Margin-bottom:10px;color:#00c0cb;"><span style="color:#111419">User Group Survey In Progress</span></h2>
+                                                            <p style="font-family:sans-serif;font-weight:normal;text-align:center;font-size:16px;line-height:22px;Margin-top:0;Margin-bottom:30px;color:#111419;"><span style="color:#111419">We've found from 70 responses (so far), 80% say that events and training is the main source of info for accessibility of course content. Is that true for your institution?</span></p>
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;height:60px">
+                                                            <a href="[SurveyLink]" target="_blank" style="padding-top:10px;padding-bottom:10px;padding-right:20px;padding-left:20px;background-color:#007900;font-family:sans-serif;font-weight:lighter;font-size:22px;line-height:22px;text-align:left;text-decoration: none;color:#FFFFFF;">Participate in the survey</a>
+                                                            <p style="Margin-top:30px;font-weight:normal;color:#262626;font-family:sans-serif;font-size:14px;line-height:18px;Margin-bottom:30px;">The results of this survey will help inform how Ally can align with existing resources and processes and how it can most effectively allow instructors and students to reach out for help.</p>
+                                                        </td>
+                                                    </tr>
+                                                </tbody>
+                                            </table>
+                                        </td>
+                                    </tr>
+                                </tbody>
+                            </table>
+
+                            <table width="600" style="border-spacing:0;Margin:0 auto;">
+                                <tbody>
+                                    <tr>
+                                        <td style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;background-color:#eaecec;width:1px;">​</td>
```

Comment:
Is this for spacing? It doesn't appear to have any effect in GMail + Chrome, but it could be having an effect in other email browsers.

If we need to keep it, we should consider making the spacer a table of its own and adding role="presentation" to help screen readers skip over it. http://blog.rebelmail.com/author/mark-robbins/

On second though, making it a `div` on its own may have the same effect.